src/test/scala/org/camunda/feel/impl/builtin/BuiltinNumberFunctionTest.scala

```diff
@@ -185,6 +185,16 @@ class BuiltinNumberFunctionsTest
     eval(" modulo(12, 5) ") should be(ValNumber(2))
   }
 
+  it should "return the negative reminder of the division of dividend by divisor" in {
+
+    eval(" modulo(-12, 5) ") should be(ValNumber(-2))
+  }
+
+  it should "return the positive reminder of the division of dividend by divisor" in {
+
+    eval(" modulo(12, -5) ") should be(ValNumber(2))
```

Comment:
Test cases from the spec:

```suggestion
    eval(" modulo(-12, 5) ") should be(ValNumber(3))
    eval(" modulo(-10.1, 4.5) ") should be(ValNumber(3.4))
```